build.sbt

```diff
@@ -7,11 +7,11 @@ val repositoryName = "prinz"
 
 // Dependency versions
 val scalaV = "2.12.10"
-val nussknackerV = "1.0.0"
+val nussknackerV = "1.2.0-staging-2021-12-09-5983-f15ac11992e3a82d32f651ef4e05b5d9458f46aa-SNAPSHOT"
```

Comment:
modify also the prinz version from 1.0.0-SNAPSHOT to 1.2.0-STAGING in all places in project